docs/content/usage/use-cases/network-segmentation.md

```diff
@@ -8,42 +8,34 @@ weight: 4
 
 ## Overview 
 
-Segmentation is a method of creating secure zones in data centers and cloud deployments that allows companies to 
-isolate workloads from one another and secure them individually, typically using policies. A useful way to test 
-the effectiveness of your segmentation is to ensure that your network segments are properly separated, e,g, your 
-Development is separated from your Production, your applications are separated from one another etc. Use the 
-Infection Monkey to verify that your network segmentation is configured properly. This way you make sure that 
-even if a certain attacker has breached your defenses, it can't move laterally between segments.
+Segmentation is a method of creating secure zones in data centers and cloud deployments that allows organizations to isolate workloads from one another and secure them individually, typically using policies. A useful way to test your company's segmentation effectiveness is to ensure that your network segments are properly separated (e.g., your development environment is isolated from your production environment and your applications are isolated from one another).
+
+[Segmentation is key](https://www.guardicore.com/use-cases/micro-segmentation/) to protecting your network, reducing the attack surface and minimizing the damage a breach can cause.
```

Comment:
```suggestion
[Segmentation is key](https://www.guardicore.com/use-cases/micro-segmentation/) to protecting your network. It can reduce the network's attack surface and minimize the damage caused during a breach.
```